src/test/java/edu/kis/vh/nursery/RyhmersJUnitTest.java

```diff
@@ -38,13 +38,13 @@ public void testIsFull() {
 		}
 
 		boolean result = ryhmer.isFull();
-		Assert.assertEquals(true, result);
+		Assert.assertEquals(false, result);

```

Comment:
Is it a correct approach to fixing errors showed by tests?
Errors are still within the project. STACK_CAPACITY is 12 so isFull should return true.